tests/packages/Microsoft.Bot.Components.Telephony.Tests/Integration/BatchInputTests/BatchInput_FixedLengthHappyPath.test.dialog

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "../../../tests.schema",
+  "$kind": "Microsoft.Test.Script",
+  "dialog": "BatchInput_FixedLengthBaseScenario.test",
+  "script": [
+    {
+      "$kind": "Microsoft.Test.UserConversationUpdate",
+      "membersAdded": [ "user" ]
+    },
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "1"
+    },
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "2"
+    },
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "3"
+    },
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "4"
+    },
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "5"
+    },
+    {
+      "$kind": "Microsoft.Test.UserSays",
+      "text": "9"
+    },
+    {
+      "$kind": "Microsoft.Test.AssertReplyActivity",
```

Comment:
Does this test flow work like so
 --> channel to bot - conversationUpdate 
 --> user to bot - 1 2 3 4 5 9
 --> bot to user - Enter your 6 digit phone number
 --> bot to user - Phone number

Why does the user send the phone before prompted to ?